packages/form/src/widgets/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import { Checkbox } from '@equinor/eds-core-react'
+
+const CheckboxWidget = (props: any) => {
+  const { label, value } = props
+
+  return (
+    <Checkbox
+      {...props}
+      label={label}
```

Comment:
is the label prop to Checkbox component defined two times here?
Both part of {...props} and defined on line 11

Reply:
fixed, removed label, since it will be added in ...props 👍